lib/cli/interactive-setup/index.js

```diff
@@ -10,11 +10,13 @@ const steps = {
   service: require('./service'),
   consoleLogin: require('./console-login'),
   dashboardLogin: require('./dashboard-login'),
-  consoleResolveOrg: require('./console-resolve-org'),
-  consoleSetupIamRole: require('./console-setup-iam-role'),
   dashboardSetOrg: require('./dashboard-set-org'),
   awsCredentials: require('./aws-credentials'),
   deploy: require('./deploy'),
+  consoleResolveOrg: require('./console-resolve-org'),
+  consoleSetupIamRole: require('./console-setup-iam-role'),
```

Comment:
I can double check if the old order was fine but I was seeing an issue when setting up a new integration, deploying the service, then configuring dev mode.

It took the integration service a minute to register up the new service with the integration. 

Moving the deploy step ahead of the integration set up meant that the service was already apart of the customers AWS account when the integration was being configured. This eliminated that delay so I could immediately look up the service functions in the integration inventory.

Reply:
Deploy is done only for newly created services (where it's mandatory to deploy so functions are accessible), it won't be issued if command was run in context of existing service.

Reply:
Yeah the creating a new service is the context I am referring to.

For example if you run `sls --console --enableDev` with the intent to create a new service, if we wait to deploy the service after the integration is set up we will need to add logic to ensure the new service's functions are added to the customers integration inventory before we can `enableDev` since there is usually a delay between the deployment and integration registering the new functions in the customers account with our inventory service.

Reply:
Yeah if this will cause some reporting issues or isn't something we want to tackle right now we should probably just not implement the onboarding changes for now since it isn't a requirement for this initial MVP anyway.

I can remove the onboarding flow stuff for now 👍

Reply:
I think this order should definitely remain untouched. Also it's still not clear to me why it was changed, knowing that _deploy_ runs only for new services, which cannot be integrated upfront for dev mode anyway